src/components/SendModalComponent.vue

```diff
@@ -42,6 +45,21 @@ onMounted(async () => {
 });
 
 async function sendAsset() {
+  // Check network and update previous values
+  let chainID = await web3.eth.getChainId();
```

Comment:
```suggestion
  let chainId = await web3.eth.getChainId();
```

Reply:
put this in a try / catch 
With network requests, we never know what can happen... let's make sure it doesn't break the app if the call fails